pkg/transaction/eventgenerator.go

```diff
@@ -78,13 +78,15 @@ func (e *Generator) trackMetrics(summaryEvent LogEvent, bytes int64) {
 		if summaryEvent.TransactionSummary.Application != nil {
 			appName = summaryEvent.TransactionSummary.Application.Name
 		}
-		teamName := ""
+		var team metric.Team
```

Comment:
the team object here is the same as in the transaction details, can we make a single type?